dts/bindings/dma/st,stm32-dma-v2bis.yaml

```diff
@@ -3,6 +3,7 @@
 
 description: |
   STM32 DMA controller (V2bis) for the stm32F1 and stm32L1 soc families
+  also for the stm32f030/f070 and stm32f0x1/f0x2/f0x8 soc families
```

Comment:
Does it mean we have F0 SoCs that don't use the same DMA hardware block

Reply:
no, it means the devices available in the stm32f0 family are stm32f030/f070  (RefMan  RM0360)  and stm32f0x1/f0x2/f0x8  (ref Man RM091) which all have DMA of that version (V2bis) (=fixed peripheral request)

Reply:
Would be nice to be able to state that this works for all F0 if this is the case.

Reply:
At this time stm32F0x value line is made of
- stm32f030x and stm32f070x
- stm32f0x1/f0x2/f0x8 
That's why this comment is for few [stm32f0](https://www.st.com/content/st_com/en/products/microcontrollers-microprocessors/stm32-32-bit-arm-cortex-mcus/stm32-mainstream-mcus/stm32f0-series.html) MCUs

Reply:
> At this time stm32F0x value line is made of

Indeed, but it is likely this will be also the case of any other F0 SoC that will be added,
and it's likely that this line won't be updated when addition is made. So let's take some
(safe) shortcut here.